### DISCUSSION TITLE
Adjust the size of the markdown editor

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#4944] [Feature Requests]
![image](https://user-images.githubusercontent.com/8030134/114222577-59c81000-99a1-11eb-862d-023a8d7b7547.png)

**Comment by milgron** [comment]
This would be a great feature. I was thinking about this today.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
